### ISSUE TITLE
[BUG] [SW] Enemies are alerted to the player through doors and windows that they should not be able to see through

### ISSUE THREAD

**Issue by ninjakitty844** [ZDoom/Raze#1092]
### Raze version

Raze 1.10.2

### Which game are you running with Raze?

Shadow Warrior

### What Operating System are you using?

Windows 11

### Have you checked that no other similar issue already exists?

- [X] I have searched and not found similar issues.

### A clear and concise description of what the bug is.

When approaching the door to enter Seppuku Station, enemies will see the player through it and play their alert sound.
Somehow, enemies can also see the player through the window used to get the riot gun, even though the player is all the way down on the ground, well below the window.

![image](https://github.com/ZDoom/Raze/assets/89247988/6b707bf6-8d4e-440c-9239-5c1f868e8b4b)
![image](https://github.com/ZDoom/Raze/assets/89247988/14c19e5d-7f6b-4c4f-ba7b-769caab27b22)

### Steps to reproduce the behaviour.

Explain how to reproduce
1. Walk by Seppuku Station door
2. Walk by Seppuku Station riot gun window
3. Enemies will be alerted even though a straight line cant be drawn from the enemies to the player (without intersecting geometry)

**Comment by markanini** [comment]
Does it behave differently in other source ports? Build games can be rather buggy...

**Comment by ninjakitty844** [comment]
> Does it behave differently in other source ports? Build games can be rather buggy...

I can confirm this is not DOS behavior and it doesn't happen in other Shadow Warrior ports either. In fact this bug didn't exist in the previous version of Raze either.

**Comment by mjr4077au** [comment]
@ninjakitty844 do you mind confirming the last known good version in your opinion as it'd help backtrack what might have gone wrong here

**Comment by ninjakitty844** [comment]
> @ninjakitty844 do you mind confirming the last known good version in your opinion as it'd help backtrack what might have gone wrong here

Raze 1.9.1 is the last official release that does not have this issue, or the other one I reported about death sounds playing at incorrect times. I am about to write a report for another bug I found, enemies bouncing up and down when shot with uzi. That bug also does not exist in 1.9.1 but exists in all releases afterwards.

**tldr;** I think 1.9.1 was the last bug-free (mostly) release for Shadow Warrior, but I have not tested dev releases only official releases (let me know if you want me to try those too)

**Comment by ninjakitty844** [comment]
I think I know what is happening and its related to my other issue about scrambled enemy sounds. 

I think the enemies are playing their alert sound instead of their ambient sound. The enemies likely are not actually being alerted to the player, they're just playing the alert sound in place of their ambient sound for some reason...

**Comment by ninjakitty844** [comment]
This issue should maybe be closed because at this point I think it's just a duplicate/extension of my other issue...
